src/model/object/entity/food/food.h

```diff
@@ -6,12 +6,9 @@
 #define EVE_FOOD_H
 
 #include "../entity.h"
-#include "../../../geometry/shape/shape.h"
 
-#include <exception>
-
-#include <experimental/random>
 #include <iostream>
+#include <stdexcept>
```

Comment:
```suggestion
```

Reply:
resolved in ea2d8921853e0ae384be30ce03ddd5ebc85a391f